src/ensembl/src/content/app/entity-viewer/components/entity-viewer-topbar/scale-switcher/ScaleSwitcher.scss

```diff
@@ -1,3 +1,25 @@
+@import 'src/styles/common';
+
 .container {
-  /* css to come later */
+  display: flex;
+}
+
+.selectedNode {
+  border-radius: 12px;
+  height: 24px;
+  padding: 1px 20px 0 20px;
+  margin-right: 24px;
+  background-color: $black;
+  color: $white;
+  text-align: center;
+  cursor: default;
```

Comment:
isn't it the default?

Reply:
It becomes a text cursor when you hoover over the text within the div, if you hoover within the div but not on the text then it behaves default. We wanted it to be default on the text as well. I have pushed the changes for the above.